voicevox_engine/downloadable_library.py

```diff
@@ -86,3 +87,16 @@ def install_library(self, library_id: str, file: BytesIO):
 
             zf.extractall(library_dir)
         return library_dir
+
+    def uninstall_library(self, library_id: str):
+        installed_libraries = self.installed_libraries()
+        if library_id not in installed_libraries.keys():
+            raise HTTPException(status_code=404, detail="指定された音声ライブラリはインストールされていません。")
```

Comment:
最近ログやエラーは詳細に書けば書くほど利便性が上がることに気づきました。
音声ライブラリーのIDも返してあげるとかどうでしょう。
```suggestion
            raise HTTPException(status_code=404, detail=f"指定された音声ライブラリ(${library_id})はインストールされていません。")
```

Reply:
確かにそのほうがいいと思います！
ちょっと他の音声ライブラリ操作関連でも同じような修正をするべきだと思うので、そのためのPRを別で作る方向で解決しようかなと思います...!

Reply:
たしかに、了解です！